src/stylesheets/components/_navigation.scss

```diff
@@ -0,0 +1,49 @@
+.app-c-navigation {
+  padding-right: $govuk-spacing-scale-3;
+  padding-left: $govuk-spacing-scale-3;
+  background-color: $app-light-grey;
+  font-family: $govuk-font-stack;
+  @include font-smoothing;
+  @include govuk-bold-19;
+
+  @include mq($until: desktop) {
+    display: none;
+  }
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    & > li {
+      box-sizing: border-box;
+      height: 53px;
```

Comment:
design or magic number?

Reply:
Design. As @36degrees commented below, we'll ask @dashouse to review the navigation to see if we could reduce what seem like magic numbers.